torch/ao/ns/fx/mappings.py

```diff
@@ -28,15 +28,12 @@ def get_base_name_to_sets_of_related_ops() -> Dict[str, Set[NSNodeTargetType]]:
         # conv modules
         set([
             nn.Conv1d,
-            nnqd.Conv1d,
```

Comment:
where are these covered? didn't find them in the module mappings in https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fx/_lower_to_native_backend.py

Reply:
oh, just saw it's deleted because it does not work yet